invenio/ext/elasticsearch/config/es_config.py

```diff
@@ -0,0 +1,162 @@
+# Configure the elasticsearch mappings for each type
```

Comment:
I was just skimming through this and I had this question: will it be possible for external modules to register their own mappings for search? As a use case, in the inspire-next overlay, we define an extended author record model and it would be nice to be able to search into the fields that we define in there. CC: @MSusik @kaplun